tests/parm/diag_table/diag_table_hrrr_stoch

```diff
@@ -0,0 +1,365 @@
+20210512.16Z.C775.32bit.non-hydro.regional
+2021 05 12 16 00 00
```

Comment:
This date is not consistent with the model start state in specified in  tests/parm/model_configure_regional_stoch.IN?

Reply:
Thanks, @junwang-noaa!  Just updated.  I don't think this header is ever actually used, but it's good to be consistent!